Rubberduck.Refactorings/EncapsulateField/EncapsulationStrategies/EncapsulateFieldStrategyBase.cs

```diff
@@ -110,7 +109,7 @@ private void InsertNewContent(IRewriteSession refactorRewriteSession, bool isPre
 
             if (isPreview)
             {
-                AddContentBlock(NewContentTypes.PostContentMessage, RubberduckUI.EncapsulateField_PreviewMarker);
+                AddContentBlock(NewContentTypes.PostContentMessage, Rubberduck.Resources.Refactorings.EncapsulateField.PreviewMarker);
```

Comment:
The file being modified here (`EncapsulateFieldStrateyBase.cs`) is no longer in the current code base.  And, there are others like this in the `EncapsulateFieldRefactoring` (it recently underwent a significant structural refactoring).  

It would seem that making sure you are working with the latest version of the code base is the most important thing to do before trying to resolve the various build/conflict issues specific to your PR's changes.

Reply:
Just did that for each of my smaller more targeted PR.